lapack_testing.py

```diff
@@ -268,7 +268,7 @@ def run_summary_test( f, cmdline, short_summary):
                 # EIG TESTS
                 cmdbase="xeigtst"+letter+" < "+dtests[0][dtest]+".in > "+dtests[2][dtest]+".out"
         if (not just_errors and not short_summary):
-            print("-->  Testing "+name+" "+dtests[1][dtest]+" [ "+cmdbase+" ]")
+            print("Testing "+name+" "+dtests[1][dtest]+"-"+cmdline, end=' ')
```

Comment:
Are we sure we want to use cmdline here? It works better with cmdbase on my end . . .